File: output/llama3.1-8b/avatar/Java/Python/codeforces_653_A.py
import sys

def main():
    n = int(input())
    a = []
    for _ in range(n):
        value = int(input())
        a.append(value)
    a.sort()
    a = list(set(a))
    found = False
    for i in range(len(a)):
        if i + 1 < len(a) and i + 2 < len(a):
            if a[i] + 1 == a[i+1] and a[i+1] + 1 == a[i+2]:
                found = True
                break
    print("YES" if found else "NO")

if __name__ == "__main__":
    main()

File: output/llama3.1-8b/avatar/Python/Java/codeforces_653_A.java
import java.util.Arrays;
import java.util.Scanner;

public class codeforces_653_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        String[] input = scanner.nextLine().split(" ");
        int[] l = new int[input.length];
        for (int i = 0; i < input.length; i++) {
            l[i] = Integer.parseInt(input[i]);
        }
        Arrays.sort(l);
        for (int i : l) {
            if (Arrays.asList(l).contains(i + 1) && Arrays.asList(l).contains(i + 2)) {
                System.out.println("YES");
                break;
            }
        } else {
            System.out.println("NO");
        }
    }
}